lib/parser/parser-base.ts

```diff
@@ -363,6 +363,11 @@ export class ParserBase {
       this.consumeToken();
     }
   }
+  maybeToken(expected: OLexerToken) { // This only compares the text!
+    if (this.getToken().getLText() === expected.getLText()) {
+      this.consumeToken();
+    }
+  }
```

Comment:
wouldn't it make more sense to just replace `maybeWord`?

Reply:
If maybeWord is unused yes.